learners/reference.md

```diff
@@ -4,5 +4,25 @@ title: 'Reference'
 
 ## Glossary
 
-This is a placeholder file. Please add content here. 
+JYS: Here is a first draft of glossary made by Copilot, to be verified/edited as needed
 
+
+- **Automation**: The use of technology to perform tasks without human intervention, often to increase efficiency and reduce errors.
+- **Binary**: A file format that contains compiled code, which can be executed by a computer.
+- **Bundle**: A collection of files or resources packaged together, often for distribution or deployment.
+- **Continuous Integration (CI)**: A development practice where code changes are automatically tested and integrated into a shared repository frequently.
+- **Dataset**: A collection of data, often organized in a structured format, used for analysis or processing.
+- **Description**: A detailed explanation or summary of a package, function, or dataset, often found in documentation.
```

Comment:
this definition should be taylored to what the DESCRIPTION file is in the context of package creation